addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -20,15 +20,55 @@ import (
 )
 
 // AntreaConfigSpec defines the desired state of AntreaConfig
-type antreaConfigSpec struct {
-	InfraProvider string `yaml:"infraProvider"`
-	Antrea        antrea `yaml:"antrea,omitempty"`
+type AntreaConfigSpec struct {
+	InfraProvider string    `yaml:"infraProvider"`
+	Antrea        antrea    `yaml:"antrea,omitempty"`
+	AntreaNsx     antreaNsx `yaml:"antreaNsx,omitempty"`
 }
 
 type antrea struct {
 	AntreaConfigDataValue antreaConfigDataValue `yaml:"config,omitempty"`
 }
 
+type antreaNsx struct {
```

Comment:
should antrea_configs_exist in clusterbootstrap.yaml be updated to include these ?

Reply:
Yes, it should be in another pull request https://github.com/vmware-tanzu/tanzu-framework/pull/4098/

Reply:
sorry that I misunderstood your comments. because this is not default setting for clusterbootstrap, when user wants to use this feature, they can edit antreaConfig manually. @jeffwubj